amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -75,5 +76,8 @@
 }
 .table-report-new-issue {
   margin-bottom: $spacer-1;
-  margin-left: $spacer-1;
+
+  &.ml-1 {
+    margin-left: $spacer-1;
```

Comment:
Oh, nice, so we have a standard spacing amount?
How much is it?

Reply:
8px